contracts/BancorEtherToken.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.10;
```

Comment:
It's because the truffle framework expects the class `FooBar` to reside in a file called `FooBar.sol`. Can't say if it's best practice or not, but it seems pretty common.